content/docs/1.5/scalers/rabbitmq-queue.md

```diff
@@ -15,29 +15,38 @@ This specification describes the `rabbitmq` trigger for RabbitMQ Queue.
 triggers:
 - type: rabbitmq
   metadata:
-    host: RabbitMqHost
-    queueLength: '20' # Optional. Queue length target for HPA. Default: 20 messages
+    queueLength: '20'
     queueName: testqueue
     includeUnacked: 'true' # Optional, use unacked + ready messages count
-    apiHost: RabbitApiHost # Optional HTTP management API endpoint
+  authenticationRef:
+    name: keda-trigger-auth-rabbitmq-conn
 ```
 
-The `host` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.  The resolved host should follow a format like `amqp://guest:password@localhost:5672/vhost`
+**Parameter list:**
 
-`apiHost` has the similar format but for HTTP API endpoint, like `https://guest:password@localhost:443/vhostname`. Note it has optional vhost name after the host slash which will be used to scope API request.
-
-By default `includeUnacked` is `false` in this case scaler uses AMQP protocol, requires `host` and only counts messages in the queue and ignores unacked messages.
-If `includeUnacked` is `true` then `host` is not required but `apiHost` is required in this case scaler uses HTTP management API and counts messages in the queue + unacked messages count.
+- `queueLength`: Queue length target for HPA. Default is 20. Optional.
```

Comment:
Let's re-order this a bit more logically so that it's using `queueName`, `queueLength` & `includeUnacked`.

Reply:
Done